devops/girder/plugins/AnnotationPlugin/upenncontrast_annotation/server/api/property.py

```diff
@@ -119,28 +122,70 @@ def update(self, property, params):
     )
     def find(self, params):
```

Comment:
This function does not use the `params` anymore, so its name is confusing regarding the find functions of the other models.
Why not renaming it to something like `getAllProperties` ?

Reply:
Done!